kotlin/lib/build.gradle.kts

```
@@ -1,12 +1,17 @@
 group = "com.inscreen"
-version = "0.1.1" // Update below too
+version = "0.1.2" // Update below too
+
+application {
+    mainClass.set("meow.WoofKt")
```

Is that visible to users in any way? perhaps 
```suggestion
    mainClass.set("com.inscreen.sdk")
```?

---

This was added by mistake, will remove.